Sources/ATTNParameterValidation.m

```diff
@@ -52,7 +52,8 @@ + (void)verify1DStringDictionaryOrNil:(nullable NSDictionary *)inputValue inputN
     return;
 
   if (![inputValue isKindOfClass:[NSDictionary class]]) {
-    [NSException raise:@"Bad Identifier" format:@"%@ should be of form NSDictionary<NSString *, NSString *> *", inputName];
+    NSLog(@"Bad identifier: %@ should be of form NSDictionary<NSString *, NSString *> *", inputName);
+    return;
```

Comment:
I think we'll probably want to re-write these methods to return a bool or something to tell us whether the validation was successful & handle whether or not we create events etc at the call site? Since now these will just fail silently and cause errors downstream as opposed to stopping the flow

Reply:
I thought we wanted to just fire-and-forget--even if they're bad events-and let the backend just swallow them